modules/gateways/monero/verify.php

```diff
@@ -6,39 +6,44 @@
 ini_set('display_errors', 1);
 ini_set('display_startup_errors', 1);
 error_reporting(E_ALL);
+
+//$gatewaymodule = "monero";
+//$GATEWAY = getGatewayVariables($gatewaymodule);
+
 $invoice_id = $_POST['invoice_id'];
 $payment_id = $_POST['payment_id'];
 $amount = $_POST['amount'];
+$link = $_POST['link'];
```

Comment:
We can't change link of wallet rpc! That will be a security issue

Reply:
okay.  How to get the variable from the module config into verify.php?  I tried using getGatewayVariables() but couldnt make it work. :(